src/Dapr.Client/DaprClientBuilder.cs

```diff
@@ -111,6 +113,21 @@ public DaprClientBuilder UseGrpcChannelOptions(GrpcChannelOptions grpcChannelOpt
             return this;
         }
 
+        /// <summary>
+        /// Adds the provided <paramref name="apiToken" /> on every request to the Dapr runtime.
+        /// </summary>
+        /// <param name="apiToken">The token to be added to the request headers/>.</param>
+        /// <returns>The <see cref="DaprClientBuilder" /> instance.</returns>
+        public DaprClientBuilder UseDaprApiToken(string apiToken)
+        {
+            this.DaprApiToken = apiToken;
+            if(string.IsNullOrWhiteSpace(apiToken))
+            {
+                this.DaprApiToken = Environment.GetEnvironmentVariable(Constants.DaprApiTokenEnvironmentVariable);
```

Comment:
This isnt' great because it's impossible to 'null out' the value if you don't want what's there. We'd be better off without this logic.

Reply:
Simple and predictable works every time.